environment.py

```diff
@@ -18,14 +19,14 @@ def _step(self, action):
         result = self.gym.step(action)
         self.state, self.reward, self.done, self.info = result[:4]
         if self.random_initialization:
-            return np.float32(self.state), np.float32(self.reward), self.done
+            return np.float32(self.state['image']).reshape(1,-1), np.float32(self.reward), self.done
```

Comment:
Is it just me or there's no different  between the code in the if statement or the else statement?

Reply:
Yea these don't seem to do anything, I couldn't find how this was different before. This may need to be looked into still.

Reply:
https://github.com/itaicaspi/mgail/blob/master/environment.py#L20-L24

When there were qpos and qvel, there is difference, but I think this condition can be removed now